.editorconfig

```diff
@@ -0,0 +1,21 @@
+# EditorConfig helps developers define and maintain consistent
+# coding styles between different editors and IDEs
+# editorconfig.org
+
+root = true
+
+
+[*]
+
+# Change these settings to your own preference
```

Comment:
If we add a editorconfig file, it should match our eslint settings (it looks like this one does). Please change the comments to indicate none of the settings should be changed, otherwise contributors may switch them to something that is flagged as incorrect by eslint.